ffi/cffi-test.c

```diff
@@ -37,7 +42,7 @@ int main(int argc, char* argv[]) {
   const SnapshotHandle *snapshot_handle = snapshot_handle_res.ok;
 
   uint64_t v = version(snapshot_handle);
-  printf("version: %llu\n", v);
+  printf("version: %" PRIu64 "\n", v);
```

Comment:
Could also potentially do:
```c++
    size_t v = version(snapshot_handle);
    printf("version: %zu\n", v);
```
(technically that's C99 not C++, but I'm pretty sure both clang and gcc support it out of the box as an extension)

Reply:
I think this worked - I'm on clang FWIW

Reply:
@nicklan has been our gcc tester

Reply:
Everything breaks on MSVC :-)